packages/server/src/notifier/index.ts

```diff
@@ -1,7 +1,7 @@
 import { createNotifications } from './createNotifications'
-import { sendNotifications } from './sendNotifications'
+import { processNotifications } from './sendNotifications'
```

Comment:
Just for consistency could you either rename the function `sendNotifications` or the file `processNotifications` ?

Reply:
Done 👍